perfkitbenchmarker/pkb.py

```diff
@@ -147,6 +147,14 @@
                      'all disks together.',
                      lower_bound=1)
 
+# support for proxy
+flags.DEFINE_string('http_proxy', '',
+                    'Specify a proxy for HTTP in the form [user:passwd@]proxy.server:port.')
```

Comment:
Proxy is used on Guest VM created by Perfkit. If your cloud is deployed behind the proxy, this feature is required. Currently this feature is needed just for OpenStack but in next time if someone add support for VMWare private clouds or Azure private setups. This feature can be useful for all.